src/components/user-group/uc/deleteUserData.uc.js

```diff
@@ -32,7 +32,26 @@ const deleteUserDataFromClasses = async (userId) => {
 	return trashBinResult({ scope: 'classes', data, complete });
 };
 
-// public
-const deleteUserData = [deleteUserDataFromClasses];
+const addTeamsToTrashbinData = (teams = [], data) => {
+	const userTeams = teams.map((team) => team._id);
+	Object.assign(data, { teamIds: { userTeams } });
+};
+
+const deleteUserDataFromTeams = async (userId) => {
+	validateParams(userId);
+	debug(`deleting user mentions in teams started`, { userId });
+	let complete = true;
+	const teams = await teamsRepo.getTeamsForUser(userId);
```

Comment:
rename to teamIds

Reply:
In this code we are operating on user and his Id to get teams data, naming here is good.